packages/app-store/applecalendar/api/add.ts

```diff
@@ -18,9 +18,29 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       select: {
         email: true,
         id: true,
+        credentials: {
+          where: {
+            type: "apple_calendar",
+          },
+        },
       },
     });
 
+    let alreadyExistsId = -1;
```

Comment:
NIT: you can rename this to something like `alreadyExisingCredentialId` or `alreadyExisingCalendarCredentialId`